examples/viber-demo-bot/src/modules/getStarted/actions.ts

```diff
@@ -135,18 +135,58 @@ async function getTest8(user: InMemoryUser, payload: string) {
         .end();
 }
 
-addAction(getStartedModule, getStartedSecond);
-addTextRule(getStartedModule, getStartedSecond, /.*/);
-async function getStartedSecond(user: InMemoryUser, payload: string) {
-    console.log(payload);
-    const now = new Date();
+addAction(getStartedModule, test_tracking);
+addPostbackRule(getStartedModule, test_tracking, 'object');
+async function test_tracking(user: InMemoryUser, payload: { tracking_data: any; text: string }) {
+    console.log(payload.text);
+    console.log(payload.tracking_data);
```

Comment:
all debugging console.logs should be removed upon merge

Reply:
I was considered this consoles as part of the testing purpose of this examples.

Reply:
Add some text then to show the user what are these data that we print.